src/js/components/tasks/capture/index.tsx

```diff
@@ -0,0 +1,269 @@
+  const handleError = (
+    err: ApiError,
+    fieldErrors: { [key: string]: string },
+  ) => {
+    /* istanbul ignore else */
+    if (isMounted.current) {
+      setCapturingChanges(false);
+      if (fieldErrors.target_directory) {
+        setPageIndex(0);
+      } else if (fieldErrors.changes) {
+        setPageIndex(1);
+      } else if (fieldErrors.commit_message) {
+        setPageIndex(2);
+      }
+    }
+  };
+
+  const defaultDir = directories.source?.[0] || 'src';
+
+  const {
+    inputs,
+    errors,
+    handleInputChange,
+    setInputs,
+    handleSubmit,
+    resetForm,
+  } = useForm({
+    fields: {
+      changes: {},
+      commit_message: '',
+      target_directory: defaultDir,
+    } as CommitData,
+    objectType: OBJECT_TYPES.COMMIT,
+    url: window.api_urls.scratch_org_commit(orgId),
+    onSuccess: handleSuccess,
+    onError: handleError,
+    shouldSubscribeToObject: false,
+  });
+
+  // When directories change, update default selection
+  const defaultDirectoryRef = useRef(defaultDir);
+  useEffect(() => {
+    const prevDefaultDir = defaultDirectoryRef.current;
+    if (defaultDir !== prevDefaultDir) {
+      setInputs({
+        ...inputs,
+        target_directory: defaultDir,
+      });
+      defaultDirectoryRef.current = defaultDir;
+    }
+  }, [defaultDir, inputs, setInputs]);
+
+  const dirSelected = Boolean(inputs.target_directory);
+  const changesChecked = Object.values(inputs.changes).flat().length;
+  const hasCommitMessage = Boolean(inputs.commit_message);
+
+  const handleClose = () => {
+    toggleModal(false);
+    setPageIndex(0);
+    resetForm();
+  };
+
+  const submitChanges = (e: React.FormEvent<HTMLFormElement>) => {
+    setCapturingChanges(true);
+    handleSubmit(e);
+  };
+
+  const pages = [
+    {
+      heading: i18n.t('Select the location to capture changes'),
+      contents: (
+        <TargetDirectoriesForm
+          key="page-1-contents"
+          directories={directories}
+          inputs={inputs as CommitData}
+          errors={errors}
+          handleInputChange={handleInputChange}
+        />
+      ),
+      footer: (
+        <Button
+          key="page-1-button-1"
+          label={i18n.t('Save & Next')}
+          variant="brand"
+          onClick={nextPage}
+          disabled={!dirSelected}
+        />
+      ),
+    },
+    {
+      heading: i18n.t('Select the changes to capture'),
+      contents: (
+        <ChangesForm
+          key="page-2-contents"
+          changeset={changeset}
+          inputs={inputs as CommitData}
+          errors={errors}
+          setInputs={setInputs}
+        />
+      ),
+      footer: [
+        <Button
+          key="page-2-button-1"
+          label={i18n.t('Go Back')}
+          variant="outline-brand"
+          onClick={prevPage}
+        />,
+        // <Button
+        //   key="page-2-button-2"
+        //   label={i18n.t('Ignore Selected Changes')}
+        //   variant="outline-brand"
+        // />,
```

Comment:
I prefer removing things like this entirely to keeping them commented out; is there a compelling reason to leave this commented out?

Reply:
Again, I could go either way. I left it in since we anticipate adding it back soon -- but I'm fine removing it to.